### ISSUE TITLE
Add in-game music

### ISSUE THREAD

**Issue by brylie** [GodotGarden/contraption-game#5]
The contraption game might not feel complete without some simple musical ambiance. We should determine what type of music (e.g. MIDI or Ogg) to play during the game, create a playback mechanism, and add at least one game song (these can be broken into separate tasks).

# Task
- determine what types of multimedia formats Godot supports
- describe how music playback should work in-game
- develop a basic music playback 'scene'
- bonus: include an initial music track ([Free Cultural Works](https://freedomdefined.org/Definition) compatible license)
- bonus: allow user to toggle music on and off

**Comment by matiasbian** [comment]
Hello. I am a music composer. I can help you with it if you want

**Comment by brylie** [comment]
Yes, thanks! Do you have any works freely available, e.g. on [OpenGameArt](https://opengameart.org/)?

What types/themes of music do you think would fit well with a tinkering game? What tools/formats do you create and publish your work in?
